### ISSUE TITLE
urban net solar radiation error with a few tests on cori-knl using intel

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/E3SM#5220]
These 3 tests are failing on cori-knl using next and intel.
```
ERS.f09_f09.IELM.cori-knl_intel.elm-solar_rad
ERS.ne30pg2_r05_EC30to60E2r2.GPMPAS-JRA.cori-knl_intel.mosart-rof_ocn_2way
SMS_Ld20.f45_f45.IELMFATES.cori-knl_intel.elm-fates_eca
```

All seem to have some form of this error:
```
 82:  urban net solar radiation error: no convergence, direct beam
 82:  elm model is stopping
 82:  calling getglobalwrite with decomp_index=         1794  and elmlevel= landunit
 82:  local  landunit index =         1794
 82:  global landunit index =         1614
 82:  global gridcell index =          595
 82:  gridcell longitude    =    140.000000000000
 82:  gridcell latitude     =   -6.00000000000000
 82:  landunit type         =            7
 82:  ENDRUN:
 82:  ERROR in UrbanAlbedoMod.F90 at line 1098
```

**Comment by ndkeen** [comment]
Actually for all 3 tests, I am unable to reproduce on cori-knl using Oct 6th master or Oct 6th next.

**Comment by ndkeen** [comment]
I ran the tests several more times, but still not fails.

Ran 5 times:
`ERS.f09_f09.IELM.cori-knl_intel.elm-solar_rad`

Ran 6 times:
`SMS_Ld20.f45_f45.IELMFATES.cori-knl_intel.elm-fates_eca`

Ran 3 times:
`ERS.ne30pg2_r05_EC30to60E2r2.GPMPAS-JRA.cori-knl_intel.mosart-rof_ocn_2way`

And to try using more threads I tried this test 3 times:
`ERS_P1024x8.f09_f09.IELM.cori-knl_intel.elm-solar_rad`

**Comment by ndkeen** [comment]
https://github.com/E3SM-Project/E3SM/pull/5225 will fix the fails with DEBUG cases using 2 of these cases. Could that be a reason for the apparently intermittent fails with these OPT cases? I don't know. But as I can't seem to reproduce, I will close once the next cdash tests show passing.

**Comment by bishtgautam** [comment]
The bug that is fixed in #5225 should impact all I-compsets and I'm surprised that more of the tests aren't failing presently.

**Comment by ndkeen** [comment]
The tests are no longer failing on cdash as I would have predicted.
